### PR TITLE
Add the support for i.MX8MP

### PR DESCRIPTION
**Pull request by clementfaure**
Hello,

A small pull-request to add the i.MX8MP support.

Thanks!

### CODE REVIEW

File: core/arch/arm/plat-imx/imx-regs.h [hunk 1]
```diff
@@ -36,12 +36,13 @@
...
 #include <registers/imx7ulp.h>
-#elif defined(CFG_MX8MQ) || defined(CFG_MX8MM) || defined(CFG_MX8MN)
+#elif defined(CFG_MX8MQ) || defined(CFG_MX8MM) || defined(CFG_MX8MN) || \
+	defined(CFG_MX8MP)
 #include <registers/imx8m.h>
 #elif defined(CFG_MX8QX) || defined(CFG_MX8QM)
 #include <registers/imx8q.h>
 #else
-#error "CFG_MX6/7/7ULP or CFG_MX8MQ/8MM/8QX/8QM is not defined"
+#error "CFG_MX6/7/7ULP or CFG_MX8MQ/8MM/8MN/8MP/8QX/8QM is not defined"
```
Comment (etienne-lms) [inline core/arch/arm/plat-imx/imx-regs.h:45]: `8MN` duplicated.

**Review by etienne-lms — COMMENTED** [review]
`Acked-by: Etienne Carriere <etienne.carriere@linaro.org>`.
note comment for commit "core: imx: add support for i.MX 8MP EVK".

File: core/arch/arm/plat-imx/imx-regs.h (hunk 1 above)
Comment (clementfaure) [reply core/arch/arm/plat-imx/imx-regs.h:45]: I see only one `8MN` in this line.
This patch also adds `8MN` that was supposed to be added withc the `8MN` support.

Comment (etienne-lms) [reply core/arch/arm/plat-imx/imx-regs.h:45]: sorry, missed it. sure fine,
`MM`/`MN` confused me. thanks @jforissier for not waiting my ack :+1:

**Pull request merged**